reana_workflow_controller/workflow_run_managers/__init__.py

```diff
@@ -21,8 +22,15 @@ class WorkflowRunManager():
         {
             'name': 'SHARED_VOLUME_PATH',
             'value': '/reana'
-        }
+        },
     ]
+
+    if os.getenv('FLASK_ENV') == 'development':
+        common_env_variables.extend(({'name': 'WDB_SOCKET_SERVER',
+                                     'value': 'wdb'},
+                                    {'name': 'WDB_NO_BROWSER_AUTO_OPEN',
+                                     'value': 'True'}))
+
```

Comment:
Please insert the `common_env_variables` enrichment after the docs, i.e. the line with """Common to all workflow engines environment variables.""" should come first.